validator/rpc/standard_api.go

```diff
@@ -90,7 +90,13 @@ func (s *Server) DeleteKeystores(
 	if err != nil {
 		return nil, status.Errorf(codes.Internal, "Could not delete keys: %v", err)
 	}
-	keysToFilter := req.PublicKeys
+	// We select keys that were deleted for retrieving slashing protection history.
```

Comment:
Should we add a unit test for this?

Reply:
In description: `The Prysm delete keystores underlying logic never returns an error, so it is not possible to add a unit test for this logic, unfortunately. Nevertheless, we should still implement the expected behavior.`. So we cant add tests for this that would test this code branch, but it's important to have this logic